src/types.jl

```diff
@@ -92,3 +92,21 @@ struct FeatureTrait <: AbstractFeatureTrait end
 abstract type AbstractFeatureCollectionTrait <: AbstractTrait end
 "A FeatureCollectionTrait holds objects of `FeatureTrait` and an `extent`"
 struct FeatureCollectionTrait <: AbstractFeatureCollectionTrait end
+
+"An AbstractCRSTrait for all coordinate reference systems"
+abstract type AbstractCRSTrait end
+"An AbstractProjectedTrait for all projected coordinate reference systems"
+abstract type AbstractProjectedTrait <: AbstractCRSTrait end
+"An AbstractGeographicTrait for all geographic coordinate reference systems"
+abstract type AbstractGeographicTrait <: AbstractCRSTrait end
+"An ProjectedTrait for all projected coordinate reference systems"
```

Comment:
```suggestion
"An `AbstractCRSTrait` for all geographic coordinate reference systems"
```

Reply:
I think you have your docstrings confused (are you on mobile?)

Reply:
Yeah lack of lines breaks + mobile lol